07-routing-gossip.md

```diff
@@ -510,7 +510,7 @@ The receiving node:
   - if the `option_channel_htlc_max` bit of `message_flags` is 0:
     - MUST consider `htlc_maximum_msat` not to be present.
   - otherwise:
-    - if `htlc_maximum_msat` is not present or greater than channel capacity:
+    - if `htlc_maximum_msat` is not present, greater than channel capacity or greater than 2^32:
```

Comment:
I don't think this change is right?

Reply:
It does make sense to reject channel_updates with `htlc_maximum_msat` greater than 2^32 as this (currently) is the largest htlc possible, in fact the section above about the sending node says:
``` 
     if the htlc_maximum_msat field is present:
      ...
     MUST set htlc_maximum_msat to the maximum value it will send through this channel for a single HTLC. 
     for channels with chain_hash identifying the Bitcoin blockchain:
           MUST set this to less than 2^32.
```
Maybe is far fetched to include this change here but it's sensible for the receiving node to check the `htlc_maximum_msat` is consistent with the limits.

Reply:
I reverted the controversial change in 0c78b3d